src/wifipaf/WiFiPAFEndPoint.cpp

```diff
@@ -597,6 +602,15 @@ CHIP_ERROR WiFiPAFEndPoint::DriveSending()
         return CHIP_NO_ERROR;
     }
 
+    if (mWiFiPafLayer->mWiFiPAFTransport->WiFiPAFResourceAvailable() == false)
+    {
+        // Resource is currently unavailable, send packets later
+        StartWaitResTimer();
+        return CHIP_NO_ERROR;
+    }
+    // Reset the resource wait count
```

Comment:
This comment seems to re-state the code. Could you add things to enhance the code reading insted? Like why we reset the count, how is this count used ... or if nothing to add just remove the comment.

Reply:
Thank you for the suggestion. I will leave the comment at the definition that it may be clearer